protected/views/project/partial/check-field.php

```diff
@@ -17,11 +17,11 @@
                 <?php if (in_array($field->type, [GlobalCheckField::TYPE_TEXTAREA, GlobalCheckField::TYPE_WYSIWYG])): ?>
                     <?php $wysiwyg = in_array($field->type, [GlobalCheckField::TYPE_WYSIWYG, GlobalCheckField::TYPE_WYSIWYG_READONLY]); ?>
 
-                    <textarea class="max-width target-check-field <?= $wysiwyg ? "wysiwyg" : "" ?>"
+                    <textarea class="max-width target-check-field <?= ($wysiwyg || $field->value) ? "wysiwyg" : "" ?>"
                               rows="10"
                               id="<?= $id ?>"
                               name="<?= $name ?>"
-                        <?php if ($field->type == GlobalCheckField::TYPE_WYSIWYG_READONLY) echo "readonly"; ?>><?php echo CHtml::encode($field->value); ?></textarea>
+                        <?php if ($field->type == GlobalCheckField::TYPE_WYSIWYG_READONLY) echo "readonly"; ?>><?php echo Utils::containsSpecificHtmlSymbols($field->value); ?></textarea>
```

Comment:
I think you made a wrong replacement here - you replaced the content value instead of the class on line 20. Please test all code changes before committing, even tiny ones.